ext/lb/lb.go

```diff
@@ -461,10 +477,8 @@ func (l *LoadBalancer) isExposedContainer(id string) bool {
 		return false
 	}
 
-	log().Debugf("checking container labels: id=%s", id)
-	// ignore proxy containers
-	if _, ok := c.Config.Labels[ext.InterlockExtNameLabel]; ok {
```

Comment:
When the proxy is ignored:

1. Proxy has containers in its config
1. Kill proxy
1. Run proxy
1. Containers are _not_ written to new proxy config

When the proxy isn't ignored:

1. Proxy has containers in its config
1. Kill proxy
1. Run proxy
1. Containers are written to new proxy config